app/models/manageiq/providers/autosde/storage_manager.rb

```diff
@@ -1,7 +1,14 @@
 class ManageIQ::Providers::Autosde::StorageManager < ManageIQ::Providers::StorageManager
+  require_nested :AutosdeClient
+  require_nested :CloudVolume
+  require_nested :ClusterVolumeMapping
+  require_nested :HostInitiatorGroup
+  require_nested :HostInitiator
+  require_nested :HostVolumeMapping
+  require_nested :PhysicalStorage
```

Comment:
Without proper require_nested these constants won't be found automatically and the auto-model-class from the core persister won't work

Reply:
I followed this up with some better specs to make sure this is working properly: https://github.com/ManageIQ/manageiq-providers-autosde/pull/112